src/apps/list.js

```diff
@@ -97,14 +105,22 @@ const listApps = async (
     }
   });
 
-  return sortedCryptoApps.concat(filtered).map(app => {
-    const ins = installed.find(i => app.name === i.name);
+  const apps = sortedCryptoApps.concat(filtered);
+
+  const installed = installedList.map(({ name, hash }) => {
+    const ins = apps.find(i => name === i.name);
     return {
-      ...app,
-      installed: !!ins,
-      updated: ins ? ins.hash === app.hash : false
+      name,
+      updated: ins ? ins.hash === hash : false
```

Comment:
I think that you guys talked about changing this to `uptodate` in the call.
In case it's relevant.

Reply:
reminder for next iteration. we'll rediscuss, i'm not really annoyed by the "updated" wording personally